pkg/build/nodeimage/internal/container/docker/archive.go

```diff
@@ -215,3 +223,14 @@ func parseRepositories(data []byte) (archiveRepositories, error) {
 	}
 	return repoTags, nil
 }
+
+// return metadataEntries
```

Comment:
This line can be now removed. Also I suggest squashing commits.

Reply:
ok
```suggestion
```